src/Core/src/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -12,6 +12,10 @@ public partial class ButtonHandler : ViewHandler<IButton, UIButton>
 		// This appears to be the padding that Xcode has when "Default" content insets are used
 		public readonly static Thickness DefaultPadding = new Thickness(12, 7);
 
+		// Because we can't inherit from Button we use the container to handle
+		// Life cycle events and things like monitoring focus changed
+		public override bool NeedsContainer => true;
```

Comment:
Isn't this a big change wrapping every button now on a UIView, border, shadows etc?!

Reply:
Ish? Containers in theory shouldn't be a big change. They should be mostly unnoticeable from a behavior stand point. This is iOS as well so not as worried about performance